MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -2530,7 +2546,7 @@
             } else {
                 alignment = [_mtlDevice minimumLinearTextureAlignmentForPixelFormat: mtlFmt];
             }
-            VkFormatProperties& props = _pixelFormats.getVkFormatProperties(vk);
+            VkFormatProperties props = _pixelFormats.getVkFormatProperties(vk);
```

Comment:
Why not retrieving and using `VkFormatProperties3` here?

Reply:
Good point, it doesn't make sense in this case.